### PR TITLE
Exodii cross faction gear

### PR DESCRIPTION
**Pull request by bombasticSlacks**
#### Summary
Content "Exodii cross faction gear"

#### Purpose of change
Added some EXODII shared faction gear to make them have more connections  to the outside.

#### Describe the solution
Added an HWP barrel using 12.3ln. Just takes a few days 12.3ln isn't complicated.

Added new plate mail from Cody using Nomad gear and Platemail called Nomad Plate. It has climate control from CBMs takes the outer and normal layer, and incorporates a lycra jumpsuit, and straps (giving well distributed). Need to give Cody the blueprints and an AC + Wiring kit for this and it takes 5 weeks.

#### Additional context

image

image

image

image

image

image

### CODE REVIEW

**Comment by Drew4484** [comment]
A layer of layered kevlar as a spall liner would be a good addition to the torso if this is intended to tank rifle rounds. Adds a bit more encumbrance and makes it warmer, good thing it has AC.

File: data/json/npcs/isolated_road/isolated_road_cody_fabricate.json [hunk 1]
```diff
@@ -55,6 +60,11 @@
         "text": "[4 weeks exceptional protection] I'll wait for the best you can do.",
         "effect": [ { "u_add_var": "blacksmith_wait", "type": "number", "context": "artisans", "value": "4" } ],
         "topic": "TALK_BLACKSMITH_FABRICATE_MEASUREMENTS"
+      },
+      {
+        "text": "[5 weeks nomad armor] I want this wanderer armor.",
```
Comment (Faalagorn) [inline data/json/npcs/isolated_road/isolated_road_cody_fabricate.json:65]: [5 weeks, nomad armor]

File: data/json/npcs/isolated_road/isolated_road_cody_fabricate.json [hunk 2]
```diff
@@ -69,7 +79,15 @@
...
       "value": 3,
       "yes": "Alright so it'll be <u_val:number_artisans_blacksmith_thickness>mm armor made of the best high carbon steel a woman can scavenge.  If you're good with that I'll take the merch up front and start taking some very invasive measurements.",
-      "no": "Alright so it'll be <u_val:number_artisans_blacksmith_thickness>mm armor made with some of the best fabrication techniques the world has ever seen.  If you're good with that I'll take the merch up front and start taking some very invasive measurements."
+      "no": {
+        "u_compare_var": "blacksmith_wait",
+        "type": "number",
+        "context": "artisans",
+        "op": "==",
+        "value": 4,
+        "yes": "Alright so it'll be <u_val:number_artisans_blacksmith_thickness>mm armor made with some of the best fabrication techniques the world has ever seen.  If you're good with that I'll take the merch up front and start taking some very invasive measurements.",
```
Comment (Faalagorn) [inline data/json/npcs/isolated_road/isolated_road_cody_fabricate.json:88]: Alright, so it'll be

File: data/json/npcs/isolated_road/isolated_road_cody_fabricate.json [hunk 3]
```diff
@@ -69,7 +79,15 @@
...
+        "no": "Alright so it'll be <u_val:umber_artisans_blacksmith_thickness>mm armor made with some of the best fabrication techniques the world has ever seen as well as some help from some extradimensional friends.  I'd be lying if I said I wasn't excited.  If you're good with that I'll take the merch up front and start taking some very invasive measurements."
```
Comment (Faalagorn) [inline data/json/npcs/isolated_road/isolated_road_cody_fabricate.json:89]: Alright, so it'll be

File: data/json/npcs/isolated_road/isolated_road_cody_fabricate.json [hunk 4]
```diff
@@ -69,7 +79,15 @@
       "op": "==",
       "value": 3,
       "yes": "Alright so it'll be <u_val:number_artisans_blacksmith_thickness>mm armor made of the best high carbon steel a woman can scavenge.  If you're good with that I'll take the merch up front and start taking some very invasive measurements.",
```
Comment (Faalagorn) [inline data/json/npcs/isolated_road/isolated_road_cody_fabricate.json:81]: Alright, so it'll be

File: data/json/npcs/isolated_road/isolated_road_cody_fabricate.json (hunk 3 above)
Comment (Faalagorn) [inline data/json/npcs/isolated_road/isolated_road_cody_fabricate.json:89]: , as well as some help

**Review by Faalagorn — COMMENTED** [review]
Few typos I spotted, some are older than this PR and some might still be there, mainly missing commas.

**Pull request merged**